rfcs/0074-community-coordination-hub.md

```diff
@@ -0,0 +1,153 @@
+---
+feature: community_coordination_hub
+start-date: 2020-07-30
+author: David Arnold <dar@xoe.solutions>
+co-authors: Doron Behar <me@doronbehar.com>
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Adopt goals, spirit, concept and execution of [`kubernetes/community`]
+where it _does_ fit [us](https://nixos.org/).
+
+[`kubernetes/community`]: https://github.com/kubernetes/community
+
+# Motivation
+[motivation]: #motivation
+
+We lack an optimum level of _empowerment_ on distinct vital aspects for
+the best [leverage](https://nav.al/least). We don't have:
+
+- Efficient Management Processes, that bias the whole community towards 
+  _coordinated and optimal Resource Employment_.
+- Organizational Structure, that enacts necesary degrees of _Division of Labor_ 
+  and _Coordination_.
+- Organizational Culture, that helps to overcome the complexity bias. ([KISS](https://en.wikipedia.org/wiki/KISS_principle))
+
+# Detailed design
+[design]: #detailed-design
+
+A repository shall exist, named `NixOS/community`, which will act as central hub
+for coordinating the community's efforts.
+
+It shall comprehend, consolidate and showcase aspects of:
+
+- Communication
+  - Channels
+  - Modes & Policies
+- Governance
+  - Processes
+  - Structure
+- Onboarding
+  - Code Contributions
+  - Community Engagement
+
+It shall omit any technical subject about nix or code in general, other than
+repository tooling.
+
+To implement, myself, and anyone interested, will review https://github.com/kubernetes/community
+to the smallest details.
+
+Based on this review, we'll create a clean new repository at
+`NixOS/community`, with what we considered was worth porting from
+[Kubernetes' community portal](https://github.com/kubernetes/community).
+
+As a general guideline for this review, we'd acknowledge the success
+and impressive dynamics of the Kubernetes ecosystem.
```

Comment:
The _Detailed design_ section does not seem very detailed at all — at least, if I read this RFC as mainly proposing the creation of a new Git(Hub) repository to be used as a discussion forum.

However, if I set aside that assumption (and maybe read between the lines a bit), I see that this RFC actually seems to be proposing mainly the creation of a new team or working group, consisting of "[your]self, and anyone interested", that is to design; set up; and, at least initially, govern a new GitHub repository that would be used as a discussion forum (and, seemingly, as an archive of discussions' outcomes).  Even under that interpretation, this RFC still is missing some details that I would consider important:

1. How is the detailed design _of the repository_ to be finalized?  It seems to me that this RFC implies that the design of the `community` repository — though to be based on the team's interpretation of the Kubernetes repository — ultimately is decided in the team's sole discretion and will not go through the RFC process.  Is that the intent?

2. How are team members chosen, and how does the team make decisions?  (Majority vote? 
 Consensus?)  I suppose the RFC is fairly unambiguous that "anyone interested" may join, but it does not seem to address the latter question, and I would find being _explicit_ on both these points to seem especially important if the team needs no further approval for the final design of the repository.

Reply:
I like your observation! And it's implications for legitimacy!

> ultimately is decided in the team's sole discretion and will not go through the RFC process. Is that the intent?

Yes, the idea is to set free momentum of a kind of ad-hock working group. The idea is to avoid getting deadlocked by discussions under a RFC scrutiny.

In a draft version I actually included the idea to report back through an Annex I about the findings and detailed plan. I might want to put that paragraph back in.

So that:
- RFC get's approved (preliminary)
- Working Group constitutes and prepares Annex I
- Annex I get's submitted as an amendment to this RFC

The goal behind this strategy is to gather consensus in two steps:
1. We need to do something. Everyone ok?
2. This is what we want to do. Anyone against it?

> How are team members chosen, and how does the team make decisions? (Majority vote?
Consensus?)

I'll have to think about this! Now that you say it, I also consider specifics necessary to preserve the legitimacy chain.

Reply:
Finally addressed in https://github.com/NixOS/rfcs/pull/74/commits/f55e7ff2f8b95f35132f1ff300a2829939d10921 I think that's the cleanest way to handle this issue.